README.md

```diff
@@ -1,2 +1,38 @@
 # lake-temperature-lstm-static
-Predict lake temperatures at depth using static lake attributes
+
+Predict lake temperatures at multiple depths using an LSTM trained to make use of static lake attributes.
+
+## Installing
+
+Create a new `conda` environment by running
+
+```
+conda update -n base conda
+conda env create -f environment.yaml
+conda activate ltls
+```
+
+Note: if [Mamba](https://github.com/mamba-org/mamba) is available, the environment may be created significantly faster by running
+
+```
+mamba update -n base mamba
+mamba env create -f environment.yaml
+conda activate ltls
+```
+
+## Downloading input data
+
+For now, MNTOHA data are being used.
+A more complete set of lakes will be incorporated in the future, but MNTOHA lake data are readily available to download from ScienceBase.
+To download temperature observations, meteorological drivers, and metadata, run
+
+`snakemake -c1 -p fetch_all`
+
+Replace the `1` in `-c1` with the number of cores to execute jobs in parallel.
```

Comment:
So running jobs in parallel is that easy?! `snakemake` does it automatically if the job is allocated multiple cores?

Reply:
I haven't tried this on HPC resources, but it worked on my laptop for the download!